### PR TITLE
KFSPTS-18927 remove CU pre-encumbrance modification

### PR DESCRIPTION
**Pull request by jhulslander**
This removes all of Cornell's Pre-Encumbrance customizations.  There is a related PR in non prod SQL that does database clean up.  When reviewing this PR, please let me if you notice any additional database element that should be cleaned up

### CODE REVIEW

**Review by db483 — CHANGES_REQUESTED** [review]
I have looked at the original jiras and also looked for any code related to PreEncumbrance and I believe this removes all relevant code for the PreEncumbrance customizations.
There is just one small formatting issue on one of the constants files and I am wondering is some of those constants are still needed.

File: src/main/java/edu/cornell/kfs/fp/CuFPConstants.java [hunk 1]
```diff
@@ -110,8 +108,12 @@
         public static final String PAYER_TYPE_PR = "PR";
     }
     
-    public static class ScheduledSourceAccountingLineConstants extends PreEncumbranceSourceAccountingLineConstants {
+    public static class ScheduledSourceAccountingLineConstants {
     	public static final String SCHEDULE_TYPE = "scheduleType";
+    	public static final String END_DATE = "endDate";
```
[the remark this answers is not in the document]
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/fp/CuFPConstants.java:113]: These are used in ScheduledSourceAccountingLine, which is used on DVs and recurring DVs.  I don't see the formating issue, what looks off to you?

[the remark this answers is not in the document]
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/fp/CuFPConstants.java:113]: These are used in ScheduledSourceAccountingLine, which is used on DVs and recurring DVs. I don't see the formating issue, what looks off to you?

[the remark this answers is not in the document]
Comment (db483) [reply src/main/java/edu/cornell/kfs/fp/CuFPConstants.java:113]: Thank you for confirming the constants are still used.
public static final String START_DATE = "startDate"; and the lines after have different spacing at the beginning of the line than public static final String END_DATE = "endDate";. Maybe one uses spaces and the others tabs?

**Comment by jhulslander** [comment]
I think the formatting is fixed now

**Review by db483 — APPROVED** [review]
Changes look good.

**Pull request merged**